fastos/src/vespa/fastos/app.cpp

```diff
@@ -55,34 +54,18 @@ bool FastOS_ApplicationInterface::Init ()
 {
     bool rc=false;
 
-    if(PreThreadInit())
-    {
-        if(FastOS_Thread::InitializeClass())
-        {
-            if(FastOS_File::InitializeClass())
-            {
-                const char *errorMsg = FastOS_Socket::InitializeServices();
-
-                if(errorMsg == nullptr)
-                {
+    if(PreThreadInit()) {
+        if(FastOS_Thread::InitializeClass()) {
+            if(FastOS_File::InitializeClass()) {
+
                     _processListMutex = new std::mutex;
                     _threadPool = new FastOS_ThreadPool(128 * 1024);
                     rc = true;
-                }
-                else
-                {
-                    fprintf(stderr,
-                            "FastOS_Socket::InitializeServices() returned:\n[%s]\n",
-                            errorMsg);
-                }
-            }
-            else
+            } else
                 fprintf(stderr, "FastOS_File class initialization failed.\n");
-        }
-        else
+        } else
```

Comment:
prefer braces also for single-expression blocks

Reply:
Next time, if not remove gets there first.....